proofs/alf/cvc5/rules/Strings.smt3

```diff
@@ -261,3 +276,17 @@
              (($str_mk_re_loop_elim l (alf.add (alf.neg l) u) r1) r2))
   :conclusion (= (re.loop l u r1) r2)
 )
+
+(declare-rule str-in-re-concat-star-char ((s1 String) (s2 String :list) (r RegLan) (b Bool))
```

Comment:
I prefer the current name since it groups per `str-in-re`, not via `str-concat`. It is clearer to have all the regular expression rules have a common prefix.